website/pages/blog/nextra-4/index.mdx

```diff
@@ -0,0 +1,1315 @@
+both in same file.
+
+{/* prettier-ignore */}
+```mdx filename="✅ Front matter"
+---
+title: Foo
+description: Bar
+---
+
+{/* Will be compiled to 👇 */}
+export const metadata = {
+  title: 'Foo',
+  description: 'Bar'
+}
+```
+
+{/* prettier-ignore */}
+```mdx filename="✅ export of a metadata"
+export const metadata = {
+  title: 'Foo',
+  description: 'Bar'
+}
+```
+
+{/* prettier-ignore */}
+```mdx filename="❌ Invalid"
+---
+title: Foo
+---
+
+export const metadata = { description: 'Bar' }
+```
+
+### New Option `whiteListTagsStyling`
+
+Nextra introduces a new option called `whiteListTagsStyling`, which allows you to whitelist HTML
+elements to be replaced with components defined in the
+[`mdx-components.js` file](#use-mdx-components). By default, Nextra only replaces `<details>` and
+`<summary>`. With this option, you can extend the behavior to replace any HTML element.
+
+```js filename="next.config.mjs"
+import nextra from 'nextra'
+
+const withNextra = nextra({
+  whiteListTagsStyling: ['h1']
+})
+```
+
+{/* prettier-ignore */}
+```mdx filename="Example of replacing <h1>"
+# Hello
+
+{/* This will be replaced as well */}
+<h1>World</h1>
+```
+
+### Folders with Index Pages Changes
+
+In Nextra 2 and 3, the structure shown below is referred to as
+[folders with index pages](https://nextra.site/docs/docs-theme/page-configuration#folders-with-index-page)
+. Notice the presence of both a `docs` folder and a `docs.mdx` file at the same level:
+
+<FileTree>
+  <FileTree.Folder name="content" open>
+    <FileTree.Folder name="docs" open>
+      <FileTree.File name="getting-started.mdx" />
+    </FileTree.Folder>
+    <FileTree.File name="docs.mdx" active />
+  </FileTree.Folder>
+</FileTree>
+
+This structure is incompatible with the [page file convention](#page-file-convention), to fix it a
+new front matter option called `asIndexPage` has been introduced. You should set `asIndexPage: true`
+in the front matter of `docs/page.mdx` to achieve the same effect.
+
+<FileTree>
+  <FileTree.Folder name="app" open>
+    <FileTree.File name="layout.jsx" />
+    <FileTree.Folder name="docs" open>
+      <FileTree.File name="page.mdx" active />
+      <FileTree.Folder name="getting-started" open>
+        <FileTree.File name="page.mdx" />
+      </FileTree.Folder>
+    </FileTree.Folder>
+  </FileTree.Folder>
+</FileTree>
+
+When using the [content directory convention](#content-directory-convention), you can achieve the
+same result by setting `asIndexPage: true` in the front matter of `docs/index.mdx`
+
+<FileTree>
+  <FileTree.Folder name="content" open>
+    <FileTree.Folder name="docs" open>
+      <FileTree.File name="index.mdx" active />
+      <FileTree.File name="getting-started.mdx" />
+    </FileTree.Folder>
+  </FileTree.Folder>
+</FileTree>
+
+#### List Subpages
+
+It's now possible to automatically list all subpages of a given route as cards.
```

Comment:
Part of the code I think we should also so how it would look like in preview. You could add a screenshot alternatively just link to an example of some deployment where we can see this feature in action :)

Reply:
fixed in https://github.com/the-guild-org/website/pull/1803/commits/bdb9ff0d06bf7302190ce890f3203e8b206cea76